app/Http/Controllers/Auth/AuthenticatedSessionController.php

```diff
@@ -28,7 +28,7 @@ public function store(LoginRequest $request): RedirectResponse
 
         $request->session()->regenerate();
 
-        return redirect()->intended(route('dashboard', absolute: false));
+        return redirect()->intended(route('employee', absolute: false));
```

Comment:
please revert this to dashboard. we have implemented a dashboard for common purposes not only for employee

Reply:
this dashboard came with Laravel we don't use it for anything that's why i removed it and as we have not implemented a dashboard i changed employee to landing page @Raza9798

Reply:
> this dashboard came with Laravel which it uses as the landing page after the success of some actions we don't use it for anything that's why i removed it and as we have not implemented a dashboard so i changed employee to landing page @Raza9798

what you have build was the dashboard. that will have sidebar and modules where admin or other user roles can logged in do some operations

Reply:
But its not common....it act as a separate module we don't implement a separate common dashboard like a landing page on vispo and ncube?

and this is the dashboard i removed @Raza9798 
![image](https://github.com/user-attachments/assets/ac604d2f-d2a0-459e-81a0-e447c1310c3c)

Reply:
this was happened because you have removed the tailwind css. what you should do is you have to keep the dashboard and change the style with bootstrap

Reply:
okay then for now ill revert the files...in an new issue will style it bootstrap

Reply:
So are we going to close this PR?

Reply:
No i meant ill revert the dashboard files & related changes as you mentioned

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard

Reply:
`resources/views/layouts/app.blade.php` view cannot be deleted this should be the layout which will needs to be extend from other modules. Since this PR has many changes which are not requested I will close this PR  and make new issues for clean the project.

Reply:
Okay sure 👍